sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeries1Id.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is an array of 1 string value that uniquely identifies a Time Series instance.
+    /// </summary>
+    /// <remarks>
+    /// At the moment, Time Series Id service only supports string types to represent a TSI Id. In order to reduce the confusion on the user,
+    /// this class has been created to extend the more generic class <see cref="TimeSeriesId{T1}"/>.
+    /// </remarks>
+    public class TimeSeries1Id : TimeSeriesId<string>
+    {
+        /// <summary>
+        /// Creates a new Time Series Id with 1 string property.
```

Comment:
Creates a new instance of the <see cref="TimeSeries1Id"/> class with 1 string property.

Reply:
There's no point in adding a link to the same class you are in. The web would link you to the page you are on. Why would you want that?

Reply:
Now I think about it, existing summary is far better. 
I take that back.